deploy/fat_container/entrypoint.sh

```diff
@@ -0,0 +1,190 @@
+	if [[ -e "$live_path" ]]; then
+		echo "Existing certificate for domain $domain"
+		nginx -s reload
+		return
+	fi
+
+	echo "Creating certificate for '$domain'"
+
+	echo "Requesting Let's Encrypt certificate for '$domain'..."
+	echo "Generating OpenSSL key for '$domain'..."
+    
+	mkdir -p "$live_path" && openssl req -x509 -nodes -newkey rsa:2048 -days 1 \
+		-keyout "$live_path/privkey.pem" \
+		-out "$live_path/fullchain.pem" \
+		-subj "/CN=localhost"
+
+	echo "Reload Nginx"
+	nginx -s reload
+
+	echo "Removing key now that validation is done for $domain..."
+	rm -Rfv /etc/letsencrypt/live/$domain /etc/letsencrypt/archive/$domain /etc/letsencrypt/renewal/$domain.conf
+
+	echo "Generating certification for domain $domain"
+  mkdir "$data_path/certbot"
+	certbot certonly --webroot --webroot-path="$data_path/certbot" \
+            --register-unsafely-without-email \
+            --domains $domain \
+            --rsa-key-size $rsa_key_size \
+            --agree-tos \
+            --force-renewal
+
+	echo "Reload nginx"
+	nginx -s reload
+}
+
+configure_ssl(){
+  NGINX_SSL_CMNT="#"
+
+  echo "Generating nginx config template without domain"
+  bash "/opt/appsmith/templates/nginx_app.conf.sh" "$NGINX_SSL_CMNT" "$APPSMITH_CUSTOM_DOMAIN" > "/etc/nginx/conf.d/nginx_app.conf.template"
+
+  echo "Generating nginx configuration"
+  cat /etc/nginx/conf.d/nginx_app.conf.template | envsubst "$(printf '$%s,' $(env | grep -Eo '^APPSMITH_[A-Z0-9_]+'))" | sed -e 's|\${\(APPSMITH_[A-Z0-9_]*\)}||g' > /etc/nginx/sites-available/default
+  nginx
+  
+  if [[ -n $APPSMITH_CUSTOM_DOMAIN ]]; then
+	init_ssl_cert "$APPSMITH_CUSTOM_DOMAIN"
+  fi
+  nginx -s stop
+}
+
+configure_supervisord(){
+	SUPERVISORD_CONF_PATH="/opt/appsmith/templates/supervisord"
+	if [[ -f "/etc/supervisor/conf.d/"*.conf ]]; then
+		rm "/etc/supervisor/conf.d/"*
+	fi
+
+	cp -f "$SUPERVISORD_CONF_PATH/application_process/"*.conf /etc/supervisor/conf.d
+	if [[ "$APPSMITH_MONGODB_URI" = "mongodb://appsmith:$APPSMITH_MONGO_PASSWORD@localhost/appsmith" ]]; then
+		cp "$SUPERVISORD_CONF_PATH/mongodb.conf" /etc/supervisor/conf.d/
+	fi
+	if [[ "$APPSMITH_REDIS_URL" = "redis://127.0.0.1:6379" ]]; then
+		cp "$SUPERVISORD_CONF_PATH/redis.conf" /etc/supervisor/conf.d/
+	fi
+}
+
+echo 'Checking configuration file'
+if ! [[ -e "/opt/appsmith/configuration/docker.env" ]]; then
+	echo "Generating default configuration file"
+	AUTO_GEN_MONGO_PASSWORD=$(tr -dc A-Za-z0-9 </dev/urandom | head -c 13 ; echo '')
+	AUTO_GEN_ENCRYPTION_PASSWORD=$(tr -dc A-Za-z0-9 </dev/urandom | head -c 13 ; echo '')
+	AUTO_GEN_ENCRYPTION_SALT=$(tr -dc A-Za-z0-9 </dev/urandom | head -c 13 ; echo '')
+	bash "/opt/appsmith/templates/docker.env.sh" "$AUTO_GEN_MONGO_PASSWORD" "$AUTO_GEN_ENCRYPTION_PASSWORD" "$AUTO_GEN_ENCRYPTION_SALT" > "/opt/appsmith/configuration/docker.env"
+fi
+
+echo 'Load environment configuration'
+set -o allexport
+. /opt/appsmith/configuration/docker.env
+set +o allexport
```

Comment:
Should we check the existence of this file?

```suggestion
if [[ -f /opt/appsmith/configuration/docker.env ]]; then
  echo 'Load environment configuration'
  set -o allexport
  . /opt/appsmith/configuration/docker.env
  set +o allexport
fi
```